Ctu.h

```diff
@@ -43,10 +44,10 @@ typedef uint8_t guchar;
 typedef uint16_t gushort;
 typedef uint32_t ghandle;
 typedef uint64_t gpid;
+
 const gptr TERMADDR = 1ULL << 61;
 
 #define FOURCC(a, b, c, d) (((d) << 24) | ((c) << 16) | ((b) << 8) | (a))
-
```

Comment:
I'd rather keep this one, because the macros on either side of it are unrelated to each other.